docs/kratos/session-management/01_overview.mdx

```diff
@@ -150,36 +122,53 @@ This flow is similar to [GitHub's sudo mode](https://help.github.com/en/github/a
 The session is considered privileged when its `authenticated_at` isn't older than the `privileged_session_max_age` value defined
 in the configuration.
 
+### Configuration
+
 This is how you update the `privileged_session_max_age` to `15m`:
 
 ````mdx-code-block
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-
 <Tabs>
-  <TabItem value="cloud" label="Ory CLI" default>
+<TabItem value="console" label="Ory Console" default>
+
+To change the privileged session duration, go to [**Ory Console**](https://console.ory.sh/) → **Session Settings**, enter the desired value (Use hour (h), minute (m), second (s) to define interval, for example: 15m30s, 1m) in the Privileged Sessions section and click the **Save** button.
```

Comment:
```suggestion
To change the privileged session duration, go to [**Ory Console**](https://console.ory.sh/) → **Session Settings**, enter the desired value (Use hour (h), minute (m), second (s) to define interval, for example: 1h15m30s) in the Privileged Sessions section and click the **Save** button.
```

Reply:
maybe we should show it can be used together `1h15m30s` since you show an example of 15m below?

Reply:
added this example